.eslintrc.js

```diff
@@ -196,7 +196,7 @@ module.exports = {
         'import/no-extraneous-dependencies': [
           'error',
           {
-            devDependencies: false,
+            devDependencies: true,
```

Comment:
I don't want it to be an error to import esbuild, a devDependency. The tradeoff right is that we're at risk of importing devDependencies in src , which is how it is with every other package at the moment. At least this still makes sure it's in the package.json

Reply:
The other option would be to ts-expect-error when importing esbuild, right? How many places would that be? Might be worth it to have the extra safety this check brings.

Reply:
It would just be this file so yeah I'm up for that

Reply:
Done in https://github.com/redwoodjs/redwood/pull/9870/commits/424a08953880e4489f75e482eb9c807da02061ca